lib/process-services/src/lib/process-list/components/start-process.component.spec.ts

```diff
@@ -583,4 +587,92 @@ describe('StartFormComponent', () => {
             });
         }));
    });
+
+    describe('Select applications', () => {
+
+        beforeEach(() => {
+            fixture.detectChanges();
+            component.name = 'My new process';
+            component.showSelectApplicationDropdown = true;
+            const change = new SimpleChange(null, 3, true);
+            getDefinitionsSpy.and.returnValue(of(testMultipleProcessDefs));
+            getDeployedApplicationsSpy = spyOn(appsProcessService, 'getDeployedApplications').and.returnValue(of(deployedApps));
+            component.ngOnChanges({ 'appId': change });
+        });
+
+        it('Should be able to show application drop-down and respective process definitions if showSelectApplicationDropdown set to true', () => {
+            fixture.detectChanges();
+            const appsSelector = fixture.nativeElement.querySelector('[data-automation-id="adf-start-process-apps-drop-down"]');
+            const lableElement = fixture.nativeElement.querySelector('.adf-start-process-app-list .mat-form-field-label');
+
+            expect(appsSelector).not.toBeNull();
+            expect(lableElement.innerText).toEqual('ADF_PROCESS_LIST.START_PROCESS.FORM.LABEL.APPLICATIONS');
+
+            expect(getDeployedApplicationsSpy).toHaveBeenCalled();
+            expect(component.applications.length).toBe(6);
+
+            expect(component.selectedApplication).toEqual(deployedApps[2]);
+            expect(component.selectedApplication.id).toEqual(component.appId);
+            expect(component.selectedApplication.name).toEqual('App3');
+
+            expect(getDefinitionsSpy).toHaveBeenCalled();
+            expect(component.processDefinitions.length).toEqual(2);
+            expect(component.processDefinitions[0].name).toEqual('My Process 1');
+            expect(component.processDefinitions[1].name).toEqual('My Process 2');
+        });
+
+        it('Should able to list process-definition based on selected application', () => {
```

Comment:
Typo. Can you change it to `Should be able to` ?

Reply:
Updated